docs/desktop/gui/customwidgets.md

```diff
@@ -99,7 +99,7 @@ look like:
           this._add(control);
           break;
       }
-      return control || this.base(arguments, id);
+      return control || super(id);
```

Comment:
ok, good. This is *different* than ES6 `super`, which require that `super()` be called before `this` is available. We have much code that depends on access to `this` before the superclass method is called, so I'm really glad it didn't try to emulate ES6 exactly.